.cursor/rules/studio/empty-states/RULE.md

```diff
@@ -0,0 +1,26 @@
+---
+description: "Studio: empty state patterns (presentational vs informational vs zero-results vs missing route)"
+globs:
+  - apps/studio/**/*.{ts,tsx}
+alwaysApply: false
+---
+
+# Studio empty states
+
+Use the Design System UI pattern docs as the source of truth:
+
+- Documentation: `apps/design-system/content/docs/ui-patterns/empty-states.mdx`
+- Demos:
+  - `apps/design-system/registry/default/example/empty-state-presentational-icon.tsx`
+  - `apps/design-system/registry/default/example/empty-state-initial-state-informational.tsx`
+  - `apps/design-system/registry/default/example/empty-state-zero-items-table.tsx`
+  - `apps/design-system/registry/default/example/data-grid-empty-state.tsx`
+  - `apps/design-system/registry/default/example/empty-state-missing-route.tsx`
```

Comment:
We could start nesting examples in subfolders and just do a catch-all like:

- `apps/design-system/registry/default/example/empty-state/**`


But this will get tricky since some examples go across multiple patterns.